tox.ini

```diff
@@ -1,13 +1,15 @@
 [tox]
-envlist = py27,py33,py34,pypy,docs,pep8,py3pep8,gendoc
+envlist = {py27,py33,py34,pypy},docs,pep8,py3pep8,gendoc,coverage-report
```

Comment:
As written this has no effect.  It's a hold over from a previous attempt at Hypothesis testing.  There were going to be two sets of environments, one of which would follow a naming convention like `pyXY-unit-tests` and the other a convention like `pyXY-hypothesis`.  An easy way to express such a thing is tox's [generative syntax](https://tox.readthedocs.io/en/latest/config.html#generative-envlist).

But that would have meant that most tests got run twice, which is silly!  I elected instead to have tox always run the Hypothesis tests, except under Python 3.3, which Hypothesis no longer officially supports.